### DISCUSSION TITLE
Regarding materials for the tutorials of sscha-2023

### DISCUSSION THREAD

**Discussion by ArchSony** [SSCHAcode/python-sscha#340] [Q&A] [answered]
I am unable to find the materials which are needed for sscha-2023 tutorials. Kindly provide the materials.

**Comment by mesonepigreco** [comment]
Hi, what material exactly are you referring to? All the tutorials for the sscha-2023 have been published in the website www.sscha.eu under the tutorials. Can you specify exactly what tutorial requires materials not provided?
  All the best,
Lorenzo

**Comment by ArchSony** [comment]
Hi Sir, I was working in calculation of the electron-phonon interaction and
superconducting properties with the SSCHA. The first line given is " The
material needed for this tutorial can be downloaded here
<https://ehubox.ehu.eus/s/Y48Wc8iX9Z76jqN>.", but not available there. Also
from where I can find all the pseudopotential files and input files ( auxiliary
SSCHA dynamical matrices) ?



*Thanks and Regards*
*Archana Hota*
*Junior Research Fellow*
*School of Mineral, Metallurgical and Materials Engineering*
*Indian Institute of Technology, Bhubaneswar*
*Odisha*



On Fri, Jun 21, 2024 at 2:13 PM Lorenzo Monacelli ***@***.***>
wrote:

> Hi, what material exactly are you referring to? All the tutorials for the
> sscha-2023 have been published in the website www.sscha.eu under the
> tutorials. Can you specify exactly what tutorial requires materials not
> provided?
> All the best,
> Lorenzo
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/SSCHAcode/discussions/340#discussioncomment-9837494>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BJKNCU6EU3OVVC6ZBAJQVADZIPRU7AVCNFSM6AAAAABJTD4KVOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TQMZXGQ4TI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 
*Disclaimer: *This email and any files transmitted with it are confidential 
and intended solely for the use of the individual or entity to whom they 
are addressed. If you have received this email in error please notify the 
system manager. This message contains confidential information and is 
intended only for the individual named. If you are not the named addressee 
you should not disseminate, distribute or copy this e-mail. Please notify 
the sender immediately by e-mail if you have received this e-mail by 
mistake and delete this e-mail from your system. If you are not the 
intended recipient you are notified that disclosing, copying, distributing 
or taking any action in reliance on the contents of this information is 
strictly prohibited.

**Reply by mesonepigreco** [reply]
@ionerrea it seems the link for the school has expired. Can we get it back online? Maybe without the data for the machine, we can store it in the website space.

**Comment by ionerrea** [comment, marked as answer]
> I am unable to find the materials which are needed for sscha-2023 tutorials. Kindly provide the materials.

I have already fixed the link. It should be ok now.
